akka-remote/src/main/scala/akka/remote/artery/Association.scala

```diff
@@ -283,7 +290,19 @@ private[akka] class Association(
     }
   }
 
-  override def toString(): String =
+  // FIXME: Compression table must be owned by each channel instead of having a global one
+  private def createCompressionTable(): LiteralCompression =
+    if (transport.provider.remoteSettings.ArteryCompressionSettings.enabled) {
+      println(s"createCompressionTable for $this")
```

Comment:
right, please ignore the println and log.error's - still some raw debugging output in this PR